src/python_testing/TC_SU_2_3.py

```
@@ -0,0 +1,446 @@
+                             f"New state is {idle_event.newState} and it should be {requestorCluster.Enums.UpdateStateEnum.kIdle}")
+
+        await asyncio.sleep(5)
+
+        # Step 2: force fail the transfer mid-download, restart the provider,
+        # and observe how the DUT reacts. Per TC-SU-2.3 the DUT must EITHER
+        # resume the aborted transfer (RC[STARTOFS]=1, STARTOFS equal to the
+        # bytesDownloaded at abort) OR report a clean error / restart from
+        # zero if resume is not implemented. Both outcomes are valid; only
+        # inconsistent behavior fails the test.
+
+        #Download resumption is not supported in the SDK so the test will always
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)